### ISSUE TITLE
FR: Multiple linter settings/settings

### ISSUE THREAD

**Issue by tryonkus** [platers/obsidian-linter#1147]
## Is Your Feature Request Related to a Problem? Please Describe.

I use Longform and need slightly different Linter settings for individual notes vs compiled manuscripts. 

## Describe the Solution You'd Like

I’d like to be able to define multiple Linter rule sets. 

**Please include an example where applicable:**

When linting individual notes, I want to leave highlights alone. When linting compiled manuscripts, I want to strip highlights. (Longform already has a step to remove comments.)

Preparing a document for web publishing vs routine cleanup. 

## Describe Alternatives You've Considered

* Manually enabling and disabling Linter rules
* Running a find and replace on the compiled manuscript. 
* Using a tool like TextSoap (Mac only) to run user defined regex rule sets. This would be particularly useful for publishing preflight, where I may be making more extensive changes than normal linting.
* Running JavaScript or other scripting tool—I am able to modify scripts written by others, but I’m not experienced enough to write my own. 

## Additional Context

Feel free to hit me up for clarification. I understand you want to keep Linter focused on linting, but it makes a great preflight tool too, up to a point.

**Comment by pjkaufman** [comment]
Hey @tryonkus . Thanks for the feedback. Could you elaborate on how that stacks up against https://github.com/platers/obsidian-linter/issues/555? These two requests seem very similar to me.

**Comment by tryonkus** [comment]
Yeah, it’s basically a duplicate—I didn’t search for other requests, just the actual feature in the docs. It would be nice (not essential) to be able to save a few different rule sets for different contexts. One point zandor raises that I hadn’t thought of is that many of us have favorite tools that run only on particular platforms—Notepad++ for them, TextSoap for me. Because Obsidian+Linter runs on nearly anything, it becomes a really handy cross platform tool.Ken ===================Ken ***@***.*** +1-585-690-9653===================On Aug 24, 2024, at 10:40 AM, Peter Kaufman ***@***.***> wrote:﻿
Hey @tryonkus . Thanks for the feedback. Could you elaborate on how that stacks up against #555? These two requests seem very similar to me.
